plugins/azure-functions/api-report.md

```diff
@@ -0,0 +1,60 @@
+## API Report File for "@backstage/plugin-azure-functions"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { ApiRef } from '@backstage/core-plugin-api';
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { DiscoveryApi } from '@backstage/core-plugin-api';
+import { Entity } from '@backstage/catalog-model';
+import { FunctionsData } from '@backstage/plugin-azure-functions-common';
+import { IdentityApi } from '@backstage/core-plugin-api';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// @public (undocumented)
+export type AzureFunctionsApi = {
+  list: ({
+    functionName,
+  }: {
+    functionName: string;
+  }) => Promise<FunctionsData[]>;
+};
+
+// @public (undocumented)
+export const azureFunctionsApiRef: ApiRef<AzureFunctionsApi>;
+
+// @public (undocumented)
+export class AzureFunctionsBackendClient implements AzureFunctionsApi {
+  constructor(options: {
+    discoveryApi: DiscoveryApi;
+    identityApi: IdentityApi;
+  });
+  // (undocumented)
+  list({ functionName }: { functionName: string }): Promise<FunctionsData[]>;
```

Comment:
Returning arrays directly is discouraged, it's easier to evolve if you wrap this is array in a `AzureListFunctionsResponse` type or similarly named.